### DISCUSSION TITLE
Side panel tabs moving anti-pattern

### DISCUSSION THREAD

**Discussion by dgebel** [freeplane/freeplane#1985] [Ideas]
I love Freeplane and so glad it is still under active development. Thank you so much for all the work!

I love the side panel with the various options. However, one pain point is when the tabbed right-side panel is open but needs more than one row because the panel is narrow, selecting a tab will change the order of the tabs. This was one of Microsoft's worst UI patterns. Totally destroys muscle memory and requires you to search the list to see where the tab has gotten to, every. single. time.

Please, please turn off this "function" if at all possible!

**Comment by euu2021** [comment]
As a side note, this may be of your interest: https://github.com/EdoFro/Freeplane_TabbedPanelMod

**Comment by dpolivaev** [comment]
I thought it wasn’t possible, but I was wrong—I can disable changing the tab order for flat family looks and feels. However, I have to be honest: when an idea includes terms like ‘anti-pattern,’ it feels like a hostile jab. In this case, this ‘feature’ is implemented by the Java runtime library. I generally appreciate ideas and submissions that express your pain without any aggression toward me.

**Comment by dpolivaev** [comment]
@euu2021 How you do you like it? I used icons from @EdoFro 
<img width="284" alt="tabs" src="https://github.com/user-attachments/assets/f56a7081-56a4-483f-a862-26b1e5abc643">

**Reply by dpolivaev** [reply]
To be honest I am not sure how much I like the icons here because they are quite different from the Freeplane toolbar icons style.

**Reply by dpolivaev** [reply]
Fixed
<img width="246" alt="tabs2" src="https://github.com/user-attachments/assets/622e9ee1-bd87-48b7-9083-b7bcfecc766e">

**Reply by euu2021** [reply]
I don't know. I don't use this sidebar much, so the text labels are easier to me, because the icons require some guessing while the user don't memorize them (mainly the second one). 

I remember [this study case](https://medium.com/google-design/a-fish-in-your-ear-134deed70268) that showed that putting labels, probably, helps users finding the features.

**Comment by dgebel** [comment]
My apologies if "antipattern" came across as hostile! My understanding of the term was that it is used for common historical patterns that are (now) recommended to be avoided due to causing issues, such as programming errors, usability, etc.
My two cents, optional or at least hover tool tip labels would be nice.
